spring-cloud-skipper/spring-cloud-skipper-client/src/main/java/org/springframework/cloud/skipper/client/util/HttpClientConfigurer.java

```diff
@@ -75,12 +84,22 @@ public HttpClientConfigurer basicAuthCredentials(String username, String passwor
 	 * @return a reference to {@code this} to enable chained method invocation
 	 */
 	public HttpClientConfigurer skipTlsCertificateVerification() {
-		httpClientBuilder.setSSLContext(HttpUtils.buildCertificateIgnoringSslContext());
-		httpClientBuilder.setSSLHostnameVerifier(new NoopHostnameVerifier());
-
+		Lookup<ConnectionSocketFactory> connSocketFactoryLookup = RegistryBuilder.<ConnectionSocketFactory> create()
+			.register("https", new SSLConnectionSocketFactory(HttpUtils.buildCertificateIgnoringSslContext(), NoopHostnameVerifier.INSTANCE))
+			.register("http", new PlainConnectionSocketFactory())
+			.build();
+		httpClientBuilder.setConnectionManager(new BasicHttpClientConnectionManager(connSocketFactoryLookup));
 		return this;
 	}
-
+	private HttpClientBuilder httpClientBuilder(SSLContext sslContext) {
```

Comment:
I don't think this is currently being used in this class.

Reply:
Good catch.  I was going to extract the code from a method and make it re-usable.  Then realized that it was only being used there.   Then forgot to remove the new method.